packages/connect-thegraph-tokens/subgraph/src/aragon/aragon.ts

```diff
@@ -25,10 +25,12 @@ export function processApp(appAddress: Address, appId: string): void {
   }
 }
 
-export function processToken(tokenAddress: Address): void {
+export function processToken(tokenAddress: Address, createTemplate: boolean = true): void {
```

Comment:
The AragonInfo registry can now register a token whose template has already been created.